.github/workflows/test.yaml

```diff
@@ -14,7 +14,24 @@ env:
 jobs:
```

Comment:
could you change the commit message to sth more specific? like
```
check test coverage on CI
```
or whatever else you prefer. it just needs to be specific.

Reply:
i try this to change commit message but didn't work

Reply:
but i make my commit message for changes like this